src/drivers/net/intel.h

```diff
@@ -200,6 +200,14 @@ struct intel_descriptor {
 #define INTEL_FEXTNVM11 0x05bbcUL
 #define INTEL_FEXTNVM11_WTF	0x00002000UL	/**< Don't ask */
 
+/** I210 Packet Buffer Sizes */
+#define INTEL_E1000_RXPBS	0x02404
+#define INTEL_E1000_TXPBS	0x03404
```

Comment:
These registers do not seem to exist in the E1000.  They do exist with the same name (but different bitfield allocations) in some E1000 variants such as i350.  Also, iPXE generally does not use `E1000` within its register definitions.

I would suggest using `INTEL_RXPBS` and `INTEL_TXPBS` here.

Reply:
I've changed them to your suggestions in the most recent revision